develop/devguide/Connector/Troubleshooting/Investigating_a_protocol_thread_RTE.md

```diff
@@ -0,0 +1,64 @@
+---
+uid: Investigating_a_protocol_thread_RTE
+---
+
+# Investigating a protocol thread RTE
+
+Below, you can read how you can find the root cause when the following alarm is shown in the Alarm Console:
+
+```txt
+Thread problem in SLProtocol.exe: [DriverName/DriverVersion] ElementName – Protocolthread
+```
+
+### User skills required
```

Comment:
This line should probably start with just two '#' (i.e. ## User skills required). The linter in VS Code should give the following warning: MD001/heading-increment/header-increment: Heading levels should only increment by one level at a time [Expected: h2; Actual: h3]